### DISCUSSION TITLE
DeploymentError: File size limit exceeded (100 MB)

### DISCUSSION THREAD

**Discussion by rayisooo** [vercel/community#174] [Help] [answered]
any idea why Im getting this am I limited to only 100Mb for a project

**Comment by Pieparker** [comment, marked as answer]
This is a file size limit at GitHub. You can learn more [here](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#file-size-limits)!

**Reply by devfive** [reply]
@Pieparker are you sure it's GitHub limit? As far as I understand the docs from the link you provided it's regarding file size limit in the git repository hosted on GitHub. I am doing deployment from GH Actions and I don't think there's such limit for it. Additionally I found the exact error in Vercel CLI and it seems it's Vercel's API that sets the file size limit: https://github.com/vercel/vercel/blob/main/packages/cli/src/commands/deploy/index.ts#L714-L719

**Reply by devfive** [reply]
btw. I am doing deployment with `--archive=tgz` option (`vercel deploy --prebuilt --archive=tgz --token=$VERCEL_TOKEN`)

**Comment by RouteMars** [comment]
same issue +1
